python/ray/serve/schema.py

```diff
@@ -621,6 +621,7 @@ def get_empty_schema_dict() -> Dict:
 @dataclass
 class DeploymentStatusOverview:
     status: DeploymentStatus
+    num_running_replicas: int
```

Comment:
I named this `num_running_replicas` so it's not confused with `num_replicas` set in the config, but no strong opinion if folks think `num_replicas` is better!